cumulusci/core/flows.py

```diff
@@ -27,15 +31,16 @@ def __init__(self, project_config, flow_config, org_config, options=None, skip=N
         """ A collection of result objects in task execution order """
         self.tasks = []
         """ A collection of configured task objects, either run or failed """
-        self._init_options(options)
+        self.nested = nested
```

Comment:
can you add a docstring? i'm  pretty sure this indicates we are on an inner flow?

Reply:
Sure, you mean a docstring for `__init__`?

Reply:
just like above tasks or options

see http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html and cmd-f for "self.attr5 = None" to see how it looks i guess